docker/lr-10x/Makefile

```diff
@@ -1,16 +1,17 @@
-VERSION = 0.1.9
-TAG1 = quay.io/broad-long-read-pipelines/lr-10x:$(VERSION)
-TAG2 = quay.io/broad-long-read-pipelines/lr-10x:latest
-TAG3 = us.gcr.io/broad-dsp-lrma/lr-10x:$(VERSION)
-TAG4 = us.gcr.io/broad-dsp-lrma/lr-10x:latest
+VERSION = 0.1.18
```

Comment:
Overall, this docker deserves its own repo, with at least 1000+ lines of python code.

Reply:
Agreed - we should have a larger discussion about code organization.